docs/KinesisSource-streamingsource.md

```diff
@@ -0,0 +1,76 @@
+# Amazon kinesis spark streaming source
+
+Description
+-----------
+Spark streaming source that reads from AWS kinesis streams.
+
+Use Case
+--------
+This source is used when you want to read data from a kinesis stream in real-time. For example, you may want to read
+data from a kinesis stream write it to a cdap dataset.
+
+Properties
+----------
+**appName:** The name of the kinesis application. The application name that is used to checkpoint the Kinesis sequence
```

Comment:
@russorat Should I change the config parameter to "name" ? It is a required config parameter.